addons/web_tour/static/tests/tour_service.test.js

```diff
@@ -931,3 +931,54 @@ test("manual tour with inactive steps", async () => {
     expect(".counter .value").toHaveText("5");
     await advanceTime(10000);
 });
+
+test("automatic tour with alternative trigger", async () => {
+    expect.assertions(6);
```

Comment:
for that kind of tests, I would rather use expect.step/verifySteps

Reply:
I just want to test that tour succeed ... and not the content of console.log() ... It's quite tedious to go and change all the logs in the (other) tests when you change the content of the messages.

Reply:
no need to step the actual logged string, you can expect.step("on step x") and expect.step("step x succeeded"); (or even "on step" and "step succeeded"). If the test fails, we'll directly see that some steps are missing. With your version, we'll have a wrong number of assertions (and it's even worse with those two unnecessary expect for warn/error)